packages/teams-js/README.md

```diff
@@ -69,6 +69,10 @@ Stay tuned for examples coming soon.
 
 The [Teams Test App](https://aka.ms/teams-test-app) is used to validate the Teams client SDK APIs.
 
+## Troubleshooting
```

Comment:
Should we also add a sentence to ask people to report it on GitHub issues if they ever find it's out of date, in addition to providing the workaround?